docs/modules/deploy/pages/deploying-in-kubernetes.adoc

```diff
@@ -36,12 +36,17 @@ https://helm.sh/[Helm^] is a package manager for Kubernetes. Hazelcast is distri
 See the https://github.com/hazelcast/charts[Hazelcast Helm Charts documentation^]
 for more details.
 
+== Hazelcast Platform Operator
+
+Kubernetes Operators are software extensions to Kubernetes which help you create and manage your applications. Hazelcast Platform Operator automates common management tasks such as configuring, creating, scaling, and recovering Hazelcast clusters on Kubernetes and Red Hat OpenShift. By taking care of manual deployment and life-cycle management, Hazelcast Platform Operator makes it simpler to work with Hazelcast clusters.
```

Comment:
`tasks such as configuring, creating, scaling, and recovering`

Currently, we do not have proper recovery functionality so I think we should not mention it here. WDYT?

Reply:
Agreed 👍